Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.cpp

```diff
@@ -234,6 +242,29 @@ namespace AzToolsFramework
             return;
         }
 
+        if (descendantIndex.column() == EntityOutlinerListModel::ColumnName)
+        {
+            AZ::EntityId descendantEntityId = GetEntityIdFromIndex(descendantIndex);
+
+            // Container entities will always have overrides because they need to maintain unique positions in the scene.
+            // We are skipping checking for overrides on container entities for this reason.
+            if (!m_prefabPublicInterface->IsInstanceContainerEntity(descendantEntityId) &&
+                m_prefabOverridePublicInterface->AreOverridesPresent(descendantEntityId))
```

Comment:
nit: Which of these calls is cheaper? We could call it first for a minor performance gain.

Reply:
Good call. The IsInstanceContainerEntity will be the cheaper one because its just 2 function calls to lookup whereas the AreOverridesPresent() call needs to check 'm' components within an entity and 'n' properties within an entity to see if any overrrides exist. So the order here would be the optimized one.